shared/menubar/out-of-date.js

```diff
@@ -48,12 +57,10 @@ const styles = Styles.styleSheetCreate({
     color: Styles.globalColors.white,
   },
   boxNonCritical: {
-    backgroundColor: Styles.globalColors.lightGrey,
+    backgroundColor: Styles.globalColors.yellow,
     padding: Styles.globalMargins.tiny,
```

Comment:
Do we want to increase the top and bottom padding? Do we need to add a border radius to the top left and right corners?

Reply:
This is how the margin *should* look, without the occasional cutoff bug: 
<img width="477" alt="screen shot 2018-10-08 at 10 35 26 am" src="https://user-images.githubusercontent.com/255797/46624541-edc17400-cae5-11e8-852f-b3a03ec01c64.png">

Do you mean we should make it bigger than this?

Reply:
Nope! That looks good to me. What causes the cutoff bug?

Reply:
Haven't looked into that yet. Maybe it's part of the layout thing that John mentioned in #frontend , i.e. that the banner should be part of the blue header in layout